internal/resource/config/administrativeaccount/administrative_account_resource.go

```diff
@@ -135,26 +137,37 @@ func (r *administrativeAccountsResource) Schema(ctx context.Context, req resourc
 	id.ToSchema(&schema)
 	resp.Schema = schema
 }
-func addOptionalAdministrativeAccountFields(ctx context.Context, addRequest *client.AdministrativeAccount, plan administrativeAccountResourceModel) error {
+
+func addOptionalAdministrativeAccountFields(ctx context.Context, addRequest *client.AdministrativeAccount, plan, state administrativeAccountResourceModel) error {
 	// Empty strings are treated as equivalent to null
 	if internaltypes.IsDefined(plan.Active) {
 		addRequest.Active = plan.Active.ValueBoolPointer()
 	}
+
 	if internaltypes.IsDefined(plan.Auditor) {
 		addRequest.Auditor = plan.Auditor.ValueBoolPointer()
 	}
+
 	if internaltypes.IsDefined(plan.Department) {
 		addRequest.Department = plan.Department.ValueStringPointer()
 	}
+
 	if internaltypes.IsDefined(plan.Description) {
 		addRequest.Description = plan.Description.ValueStringPointer()
 	}
+
 	if internaltypes.IsDefined(plan.EmailAddress) {
 		addRequest.EmailAddress = plan.EmailAddress.ValueStringPointer()
 	}
-	if internaltypes.IsDefined(plan.Password) {
+
+	if internaltypes.IsDefined(plan.Password) && plan.Password.ValueString() != state.Password.ValueString() {
```

Comment:
I'm not sure this check is necessary, since we already have the RequiresReplace modifier on password

Reply:
This is needed for the Update since we require the password field. We can't use the UseStateForUnknown method due to the password field being added to the plan, and then request, which we don't want to happen.